tests/tests-cart.php

```diff
@@ -212,6 +212,7 @@ public function test_empty_cart_is_array() {
 
 		$this->assertInternalType( 'array', $cart_contents );
 		$this->assertEmpty( $cart_contents );
+		$this->assertTrue( edd_is_cart_empty() );
```

Comment:
This is an unnecessary assertion as we should be isolating the testing of the wrapper function in it's own test.